google-cloud-server/src/main/kotlin/jetbrains/buildServer/clouds/google/connector/GoogleApiConnectorImpl.kt

```diff
@@ -159,15 +159,27 @@ class GoogleApiConnectorImpl : GoogleApiConnector {
         val details = instance.image.imageDetails
         val zone = details.zone
 
-        val instanceInfo = getInstanceBuilder(instance)
-                .setMetadata(getMetadata(mutableMapOf(
+        val instanceTemplateBuilder = getInstanceTemplate(instance).toBuilder()
+        LOG.info("GCP Instance Template: $instanceTemplateBuilder")
+        LOG.info("Instance Template properties captured from GCP: ${instanceTemplateBuilder.getProperties()}")
```

Comment:
@mohsenrezaeithe, the `$instanceTemplateBuilder` and `instanceTemplateBuilder.properties` could contain sensitive information. Could you please remove these two logging lines?